yt_dlp/extractor/ted.py

```diff
@@ -26,120 +21,114 @@ class TEDIE(InfoExtractor):
         (
             (?P<type_playlist>playlists(?:/(?P<playlist_id>\d+))?) # We have a playlist
             |
-            ((?P<type_talk>talks)) # We have a simple talk
+            (?P<type_series>series) # We have a series
             |
-            (?P<type_watch>watch)/[^/]+/[^/]+
+            (?P<type_talk>talks) # We have a simple talk
         )
         (/lang/(.*?))? # The url may contain the language
-        /(?P<name>[\w-]+) # Here goes the name and then ".html"
+        /(?P<name>[\w-]+)(?:\#season_(?P<season>\d))? # Here goes the name
```

Comment:
Can you remove these comments?
They are completely unnecessary and annoying....idk how they were allowed to be merged last time